subsys/net/lib/lwm2m_client_utils/lwm2m/lwm2m_firmware.c

```diff
@@ -222,11 +228,24 @@ static void target_image_type_store(uint16_t instance, int img_type)
 
 static void reboot_work_handler(void)
 {
-#if defined(CONFIG_LWM2M_CLIENT_UTILS_FIRMWARE_UPDATE_REBOOT) &&                                   \
-	defined(CONFIG_LWM2M_CLIENT_UTILS_DEVICE_OBJ_SUPPORT) && \
-	!defined(CONFIG_ZTEST)
+#if !defined(CONFIG_ZTEST)
+	/* Call reboot execute */
+	struct lwm2m_engine_res *dev_res;
+	struct lwm2m_obj_path path;
```

Comment:
```suggestion
	if (!IS_ENABLED(CONFIG_ZTEST)) {
		/* Call reboot execute */
		struct lwm2m_engine_res *dev_res;
		struct lwm2m_obj_path path;
```

If you don't comment out the header, I believe you can use the `IS_ENABLED()` coding style.